src/App.tsx

```diff
@@ -15,35 +19,69 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const goods: Good[] = goodsFromServer.map((name, id) => ({ name, id }));
```

Comment:
It is OK to skip types sometimes - TS is able to calculate them
```suggestion
const goods = goodsFromServer.map((name, id) => ({ name, id }));
```